### ISSUE TITLE
🐛 Employment page - Editing bugs

### ISSUE THREAD

**Issue by bradystroud** [SSWConsulting/SSW.Website#1367]
**Describe the bug**

https://www.ssw.com.au/admin/index.html#/~/employment

Maybe break these into separate PBIs

```[tasklist]
### Tasks
- [x] When I edit this page, I can see there are duplicate items in Tina CMS.
- [x] When I reorder opportunities in Tina, they do not update in the UI
```

**Screenshots/Video**

<img width="1582" alt="image" src="https://github.com/SSWConsulting/SSW.Website/assets/38869720/140b3a02-2c31-4e29-9bf6-e2b8c0369199">

**Comment by Harry-Ross** [comment]
The issue with reordering was fixed by #1368

**Comment by bradystroud** [comment]
As per my conversation with @amankumarrr and @KristenHu this has already been fixed

![Image](https://github.com/SSWConsulting/SSW.Website/assets/38869720/5f2575f9-449d-46c5-a033-22bf09bedd74)
**Figure: No duplicates**
